tests/general/Issue16.java

```diff
@@ -13,9 +13,6 @@ private static void anotherCase(@AllowedAlgorithms({"AndroidKeyStore"}) String a
 
     private static void test() {
         anotherCase(ANDROID_KEY_STORE_1);
-        // False positive here.
```

Comment:
Yes this one was fixed by https://github.com/typetools/checker-framework/issues/3105

Reply:
Removing the expected error is fine if it's a false positive. But shouldn't the call stay?

Reply:
Yeah I can get your point, staying this statement here can make sure the error is fixed